lib/menu.ts

```diff
@@ -6,7 +6,7 @@
  * entirely by a custom item. For an item to be a selectable menu item, it needs `tabindex=-1`
  * attribute set. If unset, or if the "disabled" class is set, the item will not be selectable.
  *
- * Further, if `dom.data('menuItemSelected', (yesNo: boolean, elem) => {})` is set, that callback
+ * Further, if `dom.data('menuItemSelected', () => (yesNo: boolean, elem) => {})` is set, that callback
```

Comment:
I don't get this change -- was the description incorrect before?

Reply:
Yes, grainjs creates computed from simple function, resulting in below behaviour
```
dom.dataElem(elem, 'foo', () => 'bar');
dom.getData(elem, 'foo'); // => returns 'bar' NOT the callback
```
Instead to store a callback using `dom.data` you need to pass a function that returns the callback.